libensemble/libE.py

```diff
@@ -386,6 +387,13 @@ def libE_mpi_manager(mpi_comm, sim_specs, gen_specs, exit_criteria, persis_info,
     """Manager routine runs on rank 0."""
     from libensemble.comms.mpi import MainMPIComm
 
+    if Executor.executor is not None:
+        if isinstance(Executor.executor, MPIExecutor) and libE_specs["comms"] == "mpi":
```

Comment:
As inside libE_mpi_manager - I don't think need to test for comms being "mpi" here.

Reply:
Also move below logger set up (which is just below).